shopfloor/services/checkout.py

```diff
@@ -844,6 +844,10 @@ def _put_lines_in_allowed_package(self, picking, selected_lines, package):
             lines_to_pack.write(
                 {"result_package_id": package.id, "shopfloor_checkout_done": True}
             )
+            if self.work.menu.auto_post_line:
```

Comment:
I don't like that this is occurring in this method. We need to post when we reach the end of the process.
We need to keep compatibility with shopfloor_checkout_package_measurement that hooks this method to collect the package measurements. The posting should occur after collecting the measurement. You'll have to add this there also

Reply:
I've added a fixup to make sure auto_post is done afterwards, let me know if it's ok!